extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/AuthRuntimeConfig.java

```diff
@@ -76,4 +76,48 @@ public interface AuthRuntimeConfig {
      * Form Auth config
      */
     FormAuthRuntimeConfig form();
+
+    /**
+     * Require that all registered HTTP authentication mechanisms must complete the authentication.
+     * <p>
+     * Typically, this property has to be true when the credentials are carried over mTLS, when both mTLS and another
+     * authentication, for example, OIDC bearer token authentication, must succeed.
+     * In such cases, `SecurityIdentity` created by the first mechanism, mTLS, can be injected, identities created
+     * by other mechanisms will be available on `SecurityIdentity`.
+     * The mTLS mechanism is always the first mechanism, because its priority is elevated when inclusive authentication
+     * is enabled.
+     * The identities can be retrieved using utility method as in the example below:
+     *
+     * <pre>
+     * {@code
+     * io.quarkus.vertx.http.runtime.security.HttpSecurityUtils.getSecurityIdentities(securityIdentity)
+     * }
+     * </pre>
+     * <p>
+     * This property is false by default which means that the authentication process is complete as soon as the first
+     * `SecurityIdentity` is created.
+     * <p>
+     * This property will be ignored if the path specific authentication is enabled.
+     */
+    @WithDefault("false")
+    boolean inclusive();
+
+    /**
+     * By default, inclusive authentication will result in a failure if all registered HTTP authentication mechanisms
+     * do not create an identity. If you require that all registered HTTP authentication mechanisms complete
+     * the authentication, but not all of them must create the identity, use the 'lax' mode.
```

Comment:
```suggestion
```

I'm sorry but I'm finding the sentence quite complex, might be simpler to describe below as you do what each mode value means and also refer to this property from the boolean property above